content/features/00-graph-overview.adoc

```diff
@@ -3,10 +3,10 @@ title: "Graph"
 date: 2018-06-20T19:04:38+02:00
 draft: false
 image: features/graph-overview.png
-subTitle: "See how your microservices are connected in the Kiali graph"
+subTitle: "Gain insight into how your mesh is functioning"
 weight: -1
 ---
 
-The graph view allows you to select a time period and see the **requests per second** between your microservices, the **response time** of those requests and whether they were secured with **mutual TLS**.
+The graph provides a powerful way to be able to visualize what is actually happening within your service mesh. This allows you to know what components are **communicating** with each other, to see where **errors** are occurring and how **healthy** your components are, to know if where things like routerules or circuit breakers are **configured**, and to understand the rate of your **traffic flow** and other **metrics**.
```

Comment:
The sentence it too long imo.
Also, this does not sound like correct English "to know if where things like routerules"